cmd/osbuild-composer/composer.go

```diff
@@ -105,7 +105,7 @@ func (c *Composer) InitWeldr(repoPaths []string, weldrListener net.Listener,
 }
 
 func (c *Composer) InitAPI(cert, key string, enableTLS bool, enableMTLS bool, enableJWT bool, l net.Listener) error {
-	c.api = cloudapi.NewServer(c.workers, c.rpm, c.distros)
+	c.api = cloudapi.NewServer(c.workers, c.rpm, c.distros, c.config.Koji.AWS.Bucket)
```

Comment:
Taking a configuration value from Koji API configuration to create a Cloud API server does not seem correct to me.